components/brave_wallet/browser/solana_transaction_unittest.cc

```diff
@@ -515,4 +508,35 @@ TEST_F(SolanaTransactionUnitTest, GetBase64EncodedMessage) {
   EXPECT_EQ(result, base::Base64Encode(*serialized_msg));
 }
 
+TEST_F(SolanaTransactionUnitTest, GetSerializedMessage) {
+  auto tx = SolanaTransaction(kRecentBlockhash, 0, kFromAccount,
+                              {system_transfer_ins()});
+  auto expected_message_bytes = tx.message()->Serialize(nullptr);
+  ASSERT_TRUE(expected_message_bytes);
+  EXPECT_EQ(
+      tx.GetSerializedMessage(),
+      std::make_pair(*expected_message_bytes,
+                     std::vector<std::string>(
+                         {"BrG44HdsEhzapvs8bEqzvkq4egwevS3fRE6ze2ENo6S8"})));
+
+  // Create test message bytes by modifying instruction data part.
```

Comment:
Can we add a comment saying this is to create a valid message bytes which is different than `expected_message_bytes`?

Reply:
This is gone now in https://github.com/brave/brave-core/pull/14001/commits/4ee63086cdc0d66ea8b7845fceb6eecead0567ce.